scripts/file_to_c.py

```diff
@@ -26,11 +27,15 @@ def main():
     f.write("/* automatically generated */\n")
     f.write("#include <stdint.h>\n")
     f.write("#include <stddef.h>\n\n")
-
+    
     f.write("const uint8_t " + args.name + "[] = {\n")
+    
+    wordarray = inf.read()
+    b = bytearray(wordarray, 'utf-8')
```

Comment:
Why `utf-8`? Shouldn't the file be open in binary mode above?

Reply:
Yes, I chose utf-8 just to be safe because I wasn't sure about the usage of the script (if it was only in regression tests or anywhere else). But if the only use case is base64, than I agree and we can skip it!